RELEASE_PROCESS.md

```diff
@@ -236,18 +236,18 @@ spelled with complete minutiae.
 
    1. Return to the Cirrus-CI Build page for the new release tag, confirm
       (or wait for) it to complete, re-running any failed tasks as appropriate.
-   1. For anything other than an RC, download the new release artifacts from CI
-      (the binaries which were actually tested).  The items are
-      located under the *checks* tab in github for:
-
-      * `Cirrus CI / Alt Arch. Cross` - tarball for each architecture
-      * `Cirrus CI / OSX Cross` - two zip files (amd64 and arm64)
-      * `Cirrus CI / Windows Cross` - an `msi` file
-
-      Under the "Artifacts" section of each task, click the "gosrc" link,
-      find and download the release archive (`zip`, `tar.gz` or `.msi`).
-      Save the the archive with a meaningful name, for example
-      `podman-v3.0.0.msi`.
+   1. For anything other than an RC, the release artifacts need to be published along
+      with the release. These can be built locally using:
+
+      ```shell
+      $ git checkout vX.Y.Z
+      $ make podman-remote-release-darwin_amd64.zip podman-remote-release-darwin_arm64.zip podman-remote-release-windows_amd64.zip podman.msi podman-remote-static
```

Comment:
Minor nit: This results in a horizontal scroll-bar when rendered, since the line is so long.